apps/src/applab/designMode.js

```diff
@@ -882,6 +949,16 @@ designMode.configureDragAndDrop = function () {
   // element tray to the play space.
   $('#visualization').droppable({
     accept: '.new-design-element',
+    activate: function (event, ui) {
+      // Turn off clipping in app space so we can drag the element
+      // into and out of it
+      setAppSpaceClipping(false);
```

Comment:
A quick test suggests this is not necessary, presumably because the new element does not become a child of `#visualization` until after it is dropped